.github/workflows/ci_tests.yaml

```diff
@@ -0,0 +1,72 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: PyGMT Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: ${{ matrix.os }} - Python ${{ matrix.python_version }}
```

Comment:
Another typo:
```suggestion
    name: ${{ matrix.os }} - Python ${{ matrix.python-version }}
```

Reply:
I see you've been working on it on a branch in upstream `pygmt`. Should I move this Pull Request there so we can both work on it?

Reply:
Yes, that would be better.